File: web3rpc/rpc-specs/code-samples/javascript/kaia/miscellaneous/getBlsInfos.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumber = "latest";

  provider.kaia
    .getBlsInfos(blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/miscellaneous/getBlsInfos.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockNumber = 'latest'

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_bls_infos(blockNumber)

print(kaia_response)
